### PR TITLE
add storage team to pr-reminder

### PR DESCRIPTION
**Pull request by RomanBednar**

### CODE REVIEW

**Comment by RomanBednar** [comment]
Just a couple of questions regarding the reminder tool:

1) I've used our github group for a team name, is that correct? https://github.com/orgs/openshift/teams/openshift-team-storage

2) Can we expect that all team members will get pinged on slack if any PR for specified repos is open? Or is the tool smarter so it pings only members assigned to that PR?

**Comment by dobsonj** [comment]
> * I've used our github group for a team name, is that correct? https://github.com/orgs/openshift/teams/openshift-team-storage

I think this is correct, it seems the same for other teams in this config file.

/assign @jsafrane

File: clusters/app.ci/pr-reminder/pr-reminder-config.yaml [hunk 1]
```diff
@@ -71,3 +71,40 @@ data:
         repos:
           - openshift/baremetal-runtimecfg
           - openshift/kubernetes-nmstate
+      - teamMembers:
+          - fbertina
+          - hekumar
+          - jdobson
```
Comment (dobsonj) [inline clusters/app.ci/pr-reminder/pr-reminder-config.yaml:77]:
```suggestion
          - dobsonj
```

Comment (dobsonj) [reply clusters/app.ci/pr-reminder/pr-reminder-config.yaml:77]: Are these github handles or slack handles?
If github: Roman, Hemant, and Fabio have different handle names.

File: clusters/app.ci/pr-reminder/pr-reminder-config.yaml [hunk 2]
```diff
@@ -71,3 +71,40 @@ data:
...
+          - openshift/azure-file-csi-driver
+          - openshift/azure-file-csi-driver-operator
+          - openshift/cloud-provider-openstack
+          - openshift/cluster-storage-operator
+          - openshift/csi-driver-manila-operator
+          - openshift/csi-external-attacher
+          - openshift/csi-external-provisioner
+          - openshift/csi-external-resizer
+          - openshift/csi-external-snapshotter
+          - openshift/enhancements
```
Comment (dobsonj) [inline clusters/app.ci/pr-reminder/pr-reminder-config.yaml:101]: Does this mean we get reminders for _any_ PR in openshift/enhancements? Or only from our team members?

File: clusters/app.ci/pr-reminder/pr-reminder-config.yaml [hunk 3]
```diff
@@ -71,3 +71,40 @@ data:
...
+          - openshift/gcp-pd-csi-driver
+          - openshift/gcp-pd-csi-driver-operator
+          - openshift/ibm-vpc-block-csi-driver
+          - openshift/ibm-vpc-block-csi-driver-operator
+          - openshift/local-storage-operator
+          - openshift/openstack-cinder-csi-driver-operator
+          - openshift/ovirt-csi-driver-operator
+          - openshift/vmware-vsphere-csi-driver
+          - openshift/vmware-vsphere-csi-driver-operator
```
Comment (dobsonj) [inline clusters/app.ci/pr-reminder/pr-reminder-config.yaml:110]:
```suggestion
          - openshift/vmware-vsphere-csi-driver-operator
          - openshift/vsphere-problem-detector
          - openshift/ibm-vpc-node-label-updater
          - openshift/gcp-filestore-csi-driver-operator
          - openshift/gcp-filestore-csi-driver
          - openshift/csi-livenessprobe
          - openshift/csi-node-driver-registrar
          - openshift/cluster-csi-snapshot-controller-operator
```

File: clusters/app.ci/pr-reminder/pr-reminder-config.yaml [hunk 4]
```diff
@@ -71,3 +71,40 @@ data:
...
           - openshift/kubernetes-nmstate
+      - teamMembers:
+          - fbertina
+          - hekumar
+          - jdobson
+          - jsafrane
+          - rbednar
+        teamNames:
+          - openshift-team-storage
+        repos:
```
Comment (jsafrane) [inline clusters/app.ci/pr-reminder/pr-reminder-config.yaml:82]: These are missing:

csi-livenessprobe
csi-node-driver-registrar
gcp-filestore-csi-driver
gcp-filestore-csi-driver-operator
ibm-vpc-node-label-updater
sig-storage-local-static-provisioner
vsphere-problem-detector
cluster-csi-snapshot-controller-operator

File: clusters/app.ci/pr-reminder/pr-reminder-config.yaml [hunk 5]
```diff
@@ -71,3 +71,40 @@ data:
...
           - openshift/baremetal-runtimecfg
           - openshift/kubernetes-nmstate
+      - teamMembers:
+          - fbertina
+          - hekumar
+          - jdobson
+          - jsafrane
+          - rbednar
+        teamNames:
+          - openshift-team-storage
```
Comment (jsafrane) [inline clusters/app.ci/pr-reminder/pr-reminder-config.yaml:81]: Should be `storage` included too? https://github.com/orgs/openshift/teams/storage/members
That's what I use to get our team attention

File: clusters/app.ci/pr-reminder/pr-reminder-config.yaml (hunk 2 above)
Comment (jsafrane) [reply clusters/app.ci/pr-reminder/pr-reminder-config.yaml:101]: Correction, with
```
        teamNames:
          - openshift-team-storage
```
@openshift/openshift-team-storage must be tagged. I'd vote for `- storage` only

File: clusters/app.ci/pr-reminder/pr-reminder-config.yaml (hunk 1 above)
Comment (RomanBednar) [reply clusters/app.ci/pr-reminder/pr-reminder-config.yaml:77]: The [guide](https://docs.ci.openshift.org/docs/how-tos/pr-reminder/#how-to-configure-it-for-your-team) says it should be kerberos id. And according to rover your id should be `jdobson` which is also the primary email address, I suppose the other one is just an alias?

So I think it's correct how it is, or not?

Comment (jsafrane) [reply clusters/app.ci/pr-reminder/pr-reminder-config.yaml:77]: Yes, it looks correct to me.
(And we can fix it later if it's not)

File: clusters/app.ci/pr-reminder/pr-reminder-config.yaml (hunk 3 above)
Comment (RomanBednar) [reply clusters/app.ci/pr-reminder/pr-reminder-config.yaml:123]: Does this suggest to change the ordering? I've added the remaining repos and sorted alphabetically, see if the current version is ok.

File: clusters/app.ci/pr-reminder/pr-reminder-config.yaml (hunk 2 above)
Comment (RomanBednar) [reply clusters/app.ci/pr-reminder/pr-reminder-config.yaml:110]: Ack, removing `openshift-team-storage`.

File: clusters/app.ci/pr-reminder/pr-reminder-config.yaml (hunk 4 above)
Comment (RomanBednar) [reply clusters/app.ci/pr-reminder/pr-reminder-config.yaml:88]: Added.

File: clusters/app.ci/pr-reminder/pr-reminder-config.yaml (hunk 5 above)
Comment (RomanBednar) [reply clusters/app.ci/pr-reminder/pr-reminder-config.yaml]: Most likely, adding.

File: clusters/app.ci/pr-reminder/pr-reminder-config.yaml (hunk 3 above)
Comment (jsafrane) [reply clusters/app.ci/pr-reminder/pr-reminder-config.yaml:123]: I'd sort them alphabetically

**Comment by smg247** [comment]
> Just a couple of questions regarding the reminder tool:
> 
> 1. I've used our github group for a team name, is that correct? https://github.com/orgs/openshift/teams/openshift-team-storage

It's the slug for the team that you want to use in the the `openshift` org. So if you want the team that you linked it would be `openshift-team-storage`

> 2. Can we expect that all team members will get pinged on slack if any PR for specified repos is open? Or is the tool smarter so it pings only members assigned to that PR?

The tool is smart enough to only send the PR to the reviewers and assignees. If the configured team is requested to review it will get sent to everyone.

/approve
/hold because I think you will want to change the team name as mentioned above

**Comment by RomanBednar** [comment]
@smg Based on our discussion we're gonna go with `storage` team only: https://github.com/openshift/release/pull/31589#discussion_r955159910

Thank you for the update and review!

**Pull request merged**